Makefile

```
@@ -13,12 +13,14 @@
 # limitations under the License.
 
 IMG_NAME ?= kserve/rest-proxy
+# make builder configurable, default to docker.
+ENGINE ?= docker
 
 # collect args from `make run` so that they don't run twice
 ifeq (run,$(firstword $(MAKECMDGOALS)))
   RUN_ARGS := $(wordlist 2,$(words $(MAKECMDGOALS)),$(MAKECMDGOALS))
   ifneq ("$(wildcard /.dockerenv)","")
-    $(error Inside docker container, run 'make $(RUN_ARGS)')
+    $(error Inside container image, run 'make $(RUN_ARGS)')
```

In many of the ModelMesh repos, they call it the "dev", "develop" or "developer" image/container.

```suggestion
    $(error Inside the developer container, run 'make $(RUN_ARGS)')
```